### ISSUE TITLE
zig 0.11.0 NativeTargetInfo.detect function removes need for alocator

### ISSUE THREAD

**Issue by tanj** [ratfactor/ziglings#142]
```
 % zig version
0.11.0-dev.78+28288dcbb
```

```
 % zig build
C:\src\zig\ziglings\build.zig:757:20: error: expected type 'zig.CrossTarget', found 'mem.Allocator'
            builder.allocator,
            ~~~~~~~^~~~~~~~~~
C:\src\zig\zig\lib\std\mem\Allocator.zig:1:1: note: struct declared here
//! The standard memory allocation interface.
^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
C:\src\zig\zig\lib\std\zig\CrossTarget.zig:1:1: note: struct declared here
//! Contains all the same data as `Target`, additionally introducing the concept of "the native target".
^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
referenced by:
    makeInternal: C:\src\zig\ziglings\build.zig:631:34
    make: C:\src\zig\ziglings\build.zig:617:13
    remaining reference traces hidden; use '-freference-trace' to see all reference traces
```

https://ziglang.org/documentation/master/std/#root;zig.system.NativeTargetInfo.detect

The above link shows no allocator arg.

**Comment by chrboesch** [comment]
Which OS do you use?

**Comment by tanj** [comment]
Windows 10 pro

**Comment by chrboesch** [comment]
Ok, with Arch Linux and Zig version `0.10.0` or `0.11.0-dev.939+5bde627f9`, I cannot reproduce the error. Maybe a compiler issue.

**Comment by tanj** [comment]
this was fixed in pull request #136
